packages/babel-plugin-transform-react-jsx-development/test/fixtures/linux/auto-import-dev/output.mjs

```diff
@@ -0,0 +1,31 @@
+import { createElement as _createElement } from "react";
+import { jsxDEV as _jsxDEV } from "react";
+import { Fragment as _Fragment } from "react";
```

Comment:
This has the same problem as the one you mentioned for `require()`. Again, we can fix it separately.

Reply:
Sounds good. I have a workaround in my diff for now.